Makefile

```
@@ -19,6 +19,16 @@ test: | $(grpc_services_directory)
 	$(REBAR) ct
 	$(REBAR) dialyzer
 
+# This will run all ct exepct the lora suite that for now cannot be ran on MacOS
+test-mac: | $(grpc_services_directory)
+	$(REBAR) fmt --verbose --check rebar.config
+	$(REBAR) fmt --verbose --check "{src,include,test}/**/*.{hrl,erl,app.src}" --exclude-files "src/grpc/autogen/**/*"
+	$(REBAR) fmt --verbose --check "config/{test,sys}.{config,config.src}"
+	$(REBAR) xref
+	$(REBAR) eunit
+	$(REBAR) ct --suite=$(cd test/ && ls -1N *_SUITE.erl | grep -v _lorawan_ | awk '{gsub(".erl$", ""); printf("%s,", $0)}')
```

This command doesn't run with the `ls` that comes with macOS. `-N` is not a valid option

---

Grr.... I keep forgetting that macos uses an older base from FreeBSD, which doesn't support all of these flags that different Linux distros keep adding.

(The `-N` flag is to negate quoting each filename, which shouldn't be used here anyway since we don't have spaces in these filenames.)

Also, I realized that I missed a slash in the reg-ex.

```suggestion
 	$(REBAR) ct --suite=$(cd test/ && ls -1 *_SUITE.erl | grep -v _lorawan_ | awk '{gsub("\.erl$", ""); printf("%s,", $0)}')
```